Mod1.Lection2.Hw2Eugene.Task1/Mod1.Lection2.Hw2Eugene.Task1/FizzBuzz.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+var arr = new int[100];
+
+FillingArray();
+
+void FillingArray() 
+{
+    for (var i = 0; i < arr.Length; i++) 
+    {
+        arr[i] = i + 1;
+
+        if (arr[i] % 3 == 0 && arr[i] % 5 == 0)
+        {
+            Console.WriteLine($"{arr[i] = i + 1} is FizzBuzz");
```

Comment:
You re-assigned arr[i] multiple times. Not sure it is needed since you do it on line 11.